torch/_torch_docs.py

```diff
@@ -3361,6 +3361,27 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.ge`.
 """)
 
+add_docstr(torch.gradient,
+           r"""
+gradient(input, *, domain=None, dim=None, edge_order=1) -> List of Tensors
```

Comment:
"domain" seems like an imperfect word -- maybe we can find something better?

spaces_or_coords is better but very long for a kwarg

"spacing"?